adopters.md

```diff
@@ -0,0 +1,17 @@
+# Adoptors of Dragonfly
+
+Below are the adoptors of project Dragonfly. If you are using to Dragonfly to improve your distribution, please contact us with.
```

Comment:
Dragonfly are used by many companies in production. If you are using Dragonfly in your organization, we are happy to add your company logo to the list!